src/Spago/Command/Fetch.purs

```diff
@@ -250,7 +297,10 @@ run { packages, ensureRanges, isTest } = do
       LocalPackage _ -> pure unit
       WorkspacePackage _ -> pure unit
 
-  pure transitivePackages
+  pure packageDependencies
+
+getAllDependencies :: Map PackageName PackageMap -> PackageMap
```

Comment:
Uhhh I think I suggested this name, but `Fetch.getAllDependencies` has a "download things" sort of vibe. Not sure what a better name would be though, maybe just `allDependencies`? 🤔 

Then we could have a type alias `type AllDependencies = Map PackageName PackageMap` to make it extra clear

Reply:
Personally, I hate type aliases because they disappear in errors and because they more often hide the underlying structure that is informative. Would you be against defining this as a newtype?

Reply:
Newtypes are good for things that can be mixed up with other things, but I doubt we can misuse a `Map PackageName PackageMap` in the same way you'd misuse a `String`.

This is the only place where type aliases shine, because it doesn't really matter if the compiler will make it disappear, it's just there to hide the inner structure (which is a feature in this case. You'd only consume this type with the functions from this file)

Reply:
I went with `PackageTransitiveDeps` for a type alias and `toAllDependencies` due to working on the code before seeing this. Would you like me to use your naming?

Reply:
No, this is great!